services/snmp/config.go

```diff
@@ -0,0 +1,21 @@
+package snmp
+
+type Config struct {
+	// Whether Snmptrap is enabled.
+	Enabled bool `toml:"enabled"`
+	// NMS IP (Network Management Station).
+	TargetIp string `toml:"target-ip"`
+	// NMS port
+	TargetPort int `toml:"target-port"`
+	// SNMP Community
+	Community string `toml:"community"`
+	// SNMP Version
+	Version string `toml:"version"`
+	// Whether all alerts should automatically use stateChangesOnly mode.
+	// Only applies if global is also set.
+	StateChangesOnly bool `toml:"state-changes-only"`
```

Comment:
Should we also add the `Global` option like other handlers have? Or does that not work well in the context of SNMP alerts?